tests/e2e/scripts/stablePool.json

```
@@ -0,0 +1,7 @@
+{
+	"initial-deposit": "1000000stake,1000uosmo",
+	"swap-fee": "0.005",
+	"exit-fee": "0.00",
```

Hello @phamminh0811. Is it possible to do it like this?
```suggestion
	"exit-fee": "0",
```